src/elektra/redshift.ini

```diff
@@ -0,0 +1,234 @@
+; Elektra specification file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
+; This file is based on: https://github.com/ElektraInitiative/libelektra/blob/master/examples/spec/lcdproc/LCDd.ini
+;
+; A spec file "(...) defines how the configuration settings should be validated" (source: https://www.libelektra.org/tutorials/validate-configuration)
+; For details see: https://www.libelektra.org/tutorials/writing-a-specification-for-your-configuration)
+
+; Redshift uses no environment variables.
+
+[]
+mountpoint = redshift.ecf
+
+; Configuration values - section redshift
+;=======================================================
+[temp/day]
+type = unsigned_short
+description = The color temperature the screen should have during daytime.
+default = 6500
+example = 6500
+opt/long = temp-day
+opt/arg = required
+
+[temp/night]
+type = unsigned_short
+description = The color temperature the screen should have during nighttime.
+default = 4500
+example = 4500
+opt/long = temp-night
+opt/arg = required
+
+[fade]
+type = boolean
+description = Whether transitions between color temperatures (e.g. from daytime to nighttime) should happen fast or slow.
+default = 1
+example = 0
+opt = f
+opt/long = fade
+opt/arg = none
+
+[brightness]
+type = float
+description = The screen brightness.
+default = 1.0
+example = 0.8
+check/range = 0-1
+opt = b
+opt/long = brightness
+opt/arg = required
+
+[brightness/day]
+type = float
+description = The screen brightness during daytime. If both day and night brightness are set, these will overrule the value of brightness.
+default = 1.0
+example = 0.8
+check/range = 0-1
+opt/long = brightness-day
+opt/arg = required
+
+[brightness/night]
+type = float
+description = The screen brightness during nighttime. If both day and night brightness are set, these will overrule the value of brightness.
+default = 1.0
+example = 0.8
+check/range = 0-1
+opt/long = brightness-night
+opt/arg = required
+
+[gamma]
+type = string
+description = The screen gamma.
+default = 1.0:1.0:1.0
+example = 0.9:0.9:0.9
+; Regex checks that value is of format "float:float:float". Adapted from https://www.regular-expressions.info/floatingpoint.html
```

Comment:
Why not make 3 (separate) float values?

Reply:
Because I believe UX to be better with 3 values (`gamma, gamma-day, gamma-night`) than with 9 (`gamma-red, gamma-blue, gamma-green, gamma-day-red, , ..., gamma-night-blue`).

E.g. compare:
`redshift --gamma-day 1.0:1.0:1.0 --gamma-night 0.9:0.9:0.9` with  
`redshift --gamma-day-red 1.0 --gamma-day-green 1.0 --gamma-day-blue 1.0 --gamma-night-red 0.9 --gamma-night-green 0.9 --gamma-night-blue 0.9`

What's your opinion on that?

Reply:
I think it is up the redshift devs what they prefer.

Reply:
@jonls Do you have a preference?

Reply:
In your example `redshift --gamma-day 1.0:1.0:1.0` looks more reasonable. Also, currently in Redshift gamma can be specified as either a single value for all channels or three values, one for each channel. I'm not sure if what you have here would allow for a single value.

Reply:
Thanks, I will use `redshift --gamma-day 1.0:1.0:1.0` and have added support for a single value for all channels.